packages/jellyfish-api-core/__tests__/category/loan/auctionBid.test.ts

```diff
@@ -0,0 +1,412 @@
+  expect(bobLoanAcc).toStrictEqual(['1000.00000000@TSLA'])
+
+  // create DFI-TSLA
+  await bob.poolpair.create({
+    tokenA: 'DFI',
+    tokenB: 'TSLA',
+    ownerAddress: aliceColAddr
+  })
+  await bob.generate(1)
+
+  // add DFI-TSLA
+  await bob.poolpair.add({
+    a: { symbol: 'DFI', amount: 500 },
+    b: { symbol: 'TSLA', amount: 1000 }
+  })
+  await bob.generate(1)
+
+  await bob.poolpair.swap({
+    from: bobColAddr,
+    tokenFrom: 'DFI',
+    amountFrom: 600,
+    to: bobColAddr,
+    tokenTo: 'TSLA'
+  })
+  await bob.generate(1)
+  await tGroup.waitForSync()
+
+  const vaultBefore = await bob.container.call('getvault', [bobVaultId])
+  expect(vaultBefore.isUnderLiquidation).toStrictEqual(false)
+  expect(vaultBefore.collateralAmounts.length > 0).toStrictEqual(true)
+  expect(vaultBefore.loanAmount.length > 0).toStrictEqual(true)
+  expect(vaultBefore.collateralValue !== 0).toStrictEqual(true)
+  expect(vaultBefore.loanValue !== 0).toStrictEqual(true)
+
+  const auctionsBefore = await alice.container.call('listauctions')
+  expect(auctionsBefore.length).toStrictEqual(0)
+
+  // increase 10x of TSLA price
+  await alice.rpc.oracle.setOracleData(oracleId, timestamp, { prices: [{ tokenAmount: '20@TSLA', currency: 'USD' }] })
+  await alice.generate(1)
+  await tGroup.waitForSync()
+
+  // vault is liquidated now
+  const vaultAfter = await bob.container.call('getvault', [bobVaultId])
+  expect(vaultAfter.isUnderLiquidation).toStrictEqual(true)
+  expect(vaultAfter.collateralAmounts).toStrictEqual(undefined)
+  expect(vaultAfter.loanAmount).toStrictEqual(undefined)
+  expect(vaultAfter.collateralValue).toStrictEqual(undefined)
+  expect(vaultAfter.loanValue).toStrictEqual(undefined)
+
+  const auctionsAfter = await bob.container.call('listauctions')
+  expect(auctionsAfter.length > 0).toStrictEqual(true)
+  expect(auctionsAfter[0].vaultId).toStrictEqual(bobVaultId)
+  expect(auctionsAfter[0].batchCount).toStrictEqual(2)
+  expect(auctionsAfter[0].liquidationPenalty).toStrictEqual(5)
+  expect(auctionsAfter[0].batches[0].collaterals).toStrictEqual(['5000.00000000@DFI', '0.50000000@BTC'])
+  expect(auctionsAfter[0].batches[0].loan).toStrictEqual('500.00000000@TSLA')
+}
+
+beforeEach(async () => {
+  await tGroup.start()
+  await setup()
+})
+
+afterEach(async () => {
+  await tGroup.stop()
+})
+
+describe('test auctionBid', () => {
+  it('should auctionBid', async () => {
+    const auctionsBefore = await bob.container.call('listauctions')
+    expect(auctionsBefore).toStrictEqual([
+      {
+        vaultId: bobVaultId,
+        batchCount: 2,
+        liquidationPenalty: 5,
+        batches: [
+          {
+            index: 0,
+            collaterals: ['5000.00000000@DFI', '0.50000000@BTC'],
+            loan: '500.00000000@TSLA'
+          },
+          {
+            index: 1,
+            collaterals: ['5000.00000000@DFI', '0.50000000@BTC'],
+            loan: '500.00000000@TSLA'
```

Comment:
wondering what happened to the interest here?

Reply:
hmm.. yea.. it doesn't make sense